retrofit2/src/main/java/com/linecorp/armeria/client/http/retrofit2/BufferSinkHttpData.java

```diff
@@ -163,7 +165,7 @@ public boolean isEndOfStream() {
 
     @Override
     public byte[] array() {
-        return bytes;
+        return buffer.readByteArray();
```

Comment:
Cannot do that because `Buffer` is final class. https://github.com/square/okio/blob/master/okio/src/main/java/okio/Buffer.java#L50